src/sentry/conf/server.py

```diff
@@ -1666,6 +1666,10 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 )
 SENTRY_EVENT_PROCESSING_STORE_OPTIONS: dict[str, str] = {}
 
+# Transactions processing backend
```

Comment:
I think you should elaborate on this here -- specifically a sentence or 2 about the 2 ways this can be run (shared with errors vs separated)